Makefile

```diff
@@ -24,11 +24,11 @@ endif
 # Release version
 VERSION=$(shell cat VERSION)
 
-# If ENV is prod, we use VERSION for the tag, otherwise use PLATFORM
-ifeq ($(ENV), prod)
+# If ENV is from master branch, we use VERSION for the tag, otherwise use PLATFORM
```

Comment:
This will be fine. Although, https://github.com/mass-matrix/AMPL/blob/mm/modac-tests/.github/workflows/build.yml, all the ENV will need to be changed from `prod` to `master` if we want to compare ENV with master.